app/config/config.yml

```diff
@@ -1,12 +1,14 @@
 imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
-    - { resource: services.yml }
+    - { resource: ../../src/AppBundle/Resources/config/services.yml }
```

Comment:
if you want write services in the bundle, you will create Dependency Injection folder in the bundle and write the config and loader for including services.yml